files/etc/init.d/pbr

```diff
@@ -187,15 +187,7 @@ filter_options() {
 	echo "$_ret"
 	return 0
 }
-inline_set() {
-	local value="$1" inline_set i
-	for i in $value; do
-		[ "${i:0:1}" = "!" ] && i=${i:1}
-		[ "${i:0:1}" = "@" ] && i=${i:1}
-		inline_set="${inline_set:+$inline_set, }$i"
-	done
-	echo "$inline_set"
-}
+inline_set(){ set " ${1// /, } ";set "${1// [@!]/ }";set "${1# }";echo "${1% }";}
```

Comment:
Could you please elaborate on the need to insert the trailing space? Like why not `inline_set() { set " ${1// /, }"; set "${1// [@\!]/ }"; echo "${1# }"; }`.

Also, I have a concern that on bad input, any double (or triple) spaces will turn into ` ,  ,  ,` which might mess up the set. I don't know of any elegant parameter expansion/substitution to take care of that, do you?